procrastinate/manager.py

```diff
@@ -241,6 +241,116 @@ async def finish_job_by_id_async(
             delete_job=delete_job,
         )
 
+    def cancel_job_by_id(
+        self, job_id: int, abort: bool = False, delete_job: bool = False
+    ) -> bool:
+        """
+        Cancel a job by id.
+
+        Parameters
+        ----------
+        job_id : ``int``
+            The id of the job to cancel
+        abort : ``bool``
+            If True, a job in ``doing`` state will be marked as ``aborting``, but the task
+            itself has to respect the abortion request. If False, only jobs in ``todo``
+            state will be set to ``cancelled`` and won't be processed by a worker anymore.
+        delete_job : ``bool``
+            If True, the job will be deleted from the database after being cancelled. Does
+            not affect the jobs that should be aborted.
+
+        Returns
+        -------
+        ``bool``
+            True if the job to be cancelled (or to mark for abortion) was found. False if
+            no job with that ID or in a state to be cancelled was found.
```

Comment:
Sorry, the sentence is correct but I find it a bit hard to parse.
Can I suggest:
```suggestion
            If True, the job was canceled (or aborted). If False, nothing was done: either there is no job with this id or it's not in a state where it may be canceled (`todo` or `doing`)
```

WDYT?

Reply:
Absolutely, yours sounds better 🙂.

Reply:
Ok, I updated those docstrings and replaced `canceled` with `cancelled` in some other locations to be consistent. It looks like we have settled on British English.